source/options.tsx

```diff
@@ -66,14 +66,16 @@ async function init(): Promise<void> {
 			</select>
 		) as unknown as HTMLSelectElement;
 		form.before(<p>Domain selector: {dropdown}</p>, <hr/>);
-		dropdown.addEventListener('change', () => {
+		dropdown.addEventListener('change', (event) => {
 			for (const [domain, options] of optionsByDomain) {
 				if (dropdown.value === domain) {
 					options.syncForm(form);
 				} else {
 					options.stopSyncForm();
 				}
 			}
+			const newDomain = (event.target as HTMLInputElement).value;
+			select('#personal-token-link')!.setAttribute('href', `${newDomain}/settings/tokens/new?description=Refined%20GitHub&scopes=repo`);
```

Comment:
Can you just change the `.host` property as I had suggested instead? This avoids having to repeat the URL here and in the HTML

Reply:
I did not realize you could swap it like that, TIL. :)